### ISSUE TITLE
documentation for evolve using scoop incorrect

### ISSUE THREAD

**Issue by jotaigna** [bje-/NEMO#1]
The instruction to run an optimisation scenario with scoop reads:
```
$python -m scoop $(which evolve.py) -s re100
```
it should read
```
$python -m scoop $(which evolve.py -s re100)
```

**Comment by bje-** [comment]
The `$(...)` command substitution replaces the `$(...)' with the output of the command within. The `which` command returns:

    $ which evolve.py
    /usr/local/bin/evolve.py

If the `-s re100` argument is included in the `which` command, it will be lost:

    $ which evolve.py -s re100
    /usr/local/bin/evolve.py

Why do you think there is a mistake?

**Comment by jotaigna** [comment]
In my machine `which` returned an empty string, because I didn't install nemopt beforehand. What misled me to think there was a mistake is that the command`$python -m scoop` does start an instance of scoop and sits there waiting, making me think it was running NEMO. As all the interactive commands in iPython work off a git clone local directory, I thought installing NEMO via PIP was suggested for convenience and ease of use rather than a requirement. My mistake.

I can confirm that installing nemo via PIP causes `which` to return the string above and scoop running an optimisation with parallelisation (there is NEMO output in the console now). Perhaps a remark in the jupyter notebook or a stronger remark the NEMO website about this installation step would prevent others to make this mistake?

Also, the command `-s re100` was lost when trying this, but this lost to users trying this example as the default option is this very scenario. Perhaps suggesting a different scenario in the jupyter notebook example or trowing an exception unless scenarios are always specified by the user may be another instance to steer new users towards correct use of NEMO.

**Comment by bje-** [comment]
Thanks for the report. All traps for new players! I'll use the `ccgt` scenario instead.

**Comment by bje-** [comment]
Fixed by 7013e62369cbf096a0dceea5df4554a1ccb647ba. In the end, I don't think there was a need to use `which` anyway.
